DB/src/main/java/io/deephaven/db/v2/utils/TableLoggers.java

```diff
@@ -9,7 +9,6 @@
 public class TableLoggers {
 
     static {
-        UpdatePerformanceTracker.start();
         ProcessMemoryTracker.start();
```

Comment:
Doing this via static initializer feels hacky. Should we explicitly make it part of init scripts for python and groovy? Or does this happen as a side effect of those?